commands/namespaces.go

```diff
@@ -226,8 +229,15 @@ func getValidRegion(value string) string {
 // get the Namespaces that match a pattern, where the "pattern" has no wildcards but can be a
 // prefix, infix, or suffix match to a namespace ID or label.
 func getMatchingNamespaces(ctx context.Context, ss do.ServerlessService, pattern string) ([]do.OutputNamespace, error) {
+	loader := spinner.New(spinner.WithSpinner(s.Dot))
```

Comment:
nit: `s.Dot` is already the default

```suggestion
	loader := spinner.New()
```